posts/2023/july/linguist/2023-07-13-linguist.md

```diff
@@ -13,58 +13,58 @@ image:
+A common problem for all browser extensions is that they are just wrappers over google translator widget. Google's translator is good, but it is far from ideal and there are other translators which work better for some languages and use cases.
 
 
-# Meet the Linguist
+# Meet Linguist
 
-[Linguist](https://github.com/translate-tools/linguist) is all-in one translation solution, you can use full page translation, translate selected text, paste text for translate, use text to speech feature, add important translations to a dictionary to find it later if you learn languages like me.
+[Linguist](https://github.com/translate-tools/linguist) is all-in one translation solution which you can use for full page translation, translation of selected text, paste text for translate and a text to speech feature. You can also add important translations to a dictionary to find it later if you learn languages like me.
 
 ![pageTranslation.png](./pageTranslation.png)
 
 ## Use your favorite translation service
 
-Linguist have few embedded translators, include google translator, and you can add your favorite translator to linguist yourself with [custom translators](https://github.com/translate-tools/linguist/blob/bcc8c471fdede42b6bbc541144eb89a93587f605/docs/CustomTranslator.md) button. I really like this powerful feature, with Linguist you are not locked on Google Translator anymore!
+Linguist has a few embedded translators, include google translator, and you can add your favorite translator to linguist yourself with [custom translators](https://github.com/translate-tools/linguist/blob/bcc8c471fdede42b6bbc541144eb89a93587f605/docs/CustomTranslator.md) button. I really like this powerful feature - with Linguist you are not locked to using Google Translator anymore!
 
 Look, you can just insert any JavaScript code that implement translator interface:
 
 ![custom_translator.png](./custom_translator.png)
 
 ## Keep your privacy
 
-Start of version 5.0, Linguist includes embedded offline translator, based on [Bergamot project](https://github.com/browsermt/bergamot-translator) - mozilla's machine translation project, compiled to WebAssembly.
+Starting from version 5.0, Linguist includes embedded offline translator, based on [Bergamot project](https://github.com/browsermt/bergamot-translator) - mozilla's machine translation project, compiled to WebAssembly.
 
 ![settings.png](./settings.png)
 
-With offline translator, you will not sent your private texts over internet at all. Translation runs on your device locally, you still needs internet to download AI models for your languages direction once, then you can even off your internet connection and continue to translate texts.
+With the offline translator, you will not be sending your private texts over internet. The translation runs on your device locally once you have downloaded the AI models for the languages you want translate to and from.
```

Comment:
Let's leave admiration in text if this way to speak are correct. I tried to convey my delight about you no need internet "at all" after download AI models. I think it make text more expressive. And you?